cypress/e2e/share.spec.js

```diff
@@ -95,16 +95,17 @@ describe('Open test.md in viewer', function() {
 			})
 	})
 
-	it('Shares a folder as a public read only link', function() {
+	it.only('Shares a folder as a public read only link', function() {
 		cy.shareFile('/folder')
 			.then((token) => {
-				cy.logout()
-				cy.visit(`/s/${token}`)
+				return cy.logout()
+					.then(() => token)
 			})
+			.then((token) => cy.visit(`/s/${token}`))
```

Comment:
Is this change really needed?
At some point i need to learn how these thenables work in cypress.
I was under the impression that the visit worked fine before.